crypto/crypto.go

```diff
@@ -29,13 +29,16 @@ import (
 	"github.com/nuts-foundation/nuts-node/crypto/log"
 	"github.com/nuts-foundation/nuts-node/crypto/storage"
 	"path"
+	"regexp"
 )
 
 const (
 	// ModuleName contains the name of this module
 	ModuleName = "Crypto"
 )
 
+var kidPattern = regexp.MustCompile(`^[\da-zA-Z_\- :#.]+$`)
```

Comment:
package already depends on `nuts-node/crypto/storage`. Export the pattern there and use it here? (Or export from somewhere else to prevent double declaration.)

Reply:
I first implemented in the filesystem backend, then I realized it applies to Vault as well, so I copied it to there. Forgot to remove it here.

Reply:
This still needs to happen right?